azuredevops/internal/service/policy/resource_branchpolicy_auto_reviewers.go

```diff
@@ -0,0 +1,124 @@
+package policy
```

Comment:
Could you add an acceptance test for this too? :)

Reply:
Done!